arviz/stats/stats.py

```diff
@@ -97,25 +107,40 @@ def compare(dataset_dict, ic="waic", method="stacking", b_samples=1000, alpha=1,
         It's always 0 for the top-ranked model.
     warning : A value of 1 indicates that the computation of the IC may not be reliable. This could
         be indication of WAIC/LOO starting to fail see http://arxiv.org/abs/1507.04544 for details.
+    scale : Scale used for the IC.
     """
     names = list(dataset_dict.keys())
+    scale = scale.lower()
+    if scale == "log":
+        scale_value = 1
+        ascending = False
+    else:
+        if scale == "negative_log":
+            scale_value = -1
+        else:
```

Comment:
I think these conditionals should explicitly check for the documented options ('deviance', 'log', 'negative_log'), and throw a TypeError otherwise.